sign/rpmsignverity.c

```diff
@@ -0,0 +1,234 @@
+/**
+ * Copyright (C) 2020 Facebook
+ *
+ * Author: Jes Sorensen <jsorensen@fb.com>
+ */
+
+#include "system.h"
+
+#include <rpm/rpmlib.h>		/* RPMSIGTAG & related */
+#include <rpm/rpmlog.h>		/* rpmlog */
+#include <rpm/rpmfi.h>
+#include <rpm/rpmpgp.h>		/* rpmDigestLength */
+#include "lib/header.h"		/* HEADERGET_MINMEM */
+#include "lib/header_internal.h"
+#include "lib/rpmtypes.h"	/* rpmRC */
+#include <libfsverity.h>
+#include <rpmio/rpmio_internal.h>
```

Comment:
Always se "..." when including internal-only headers, like you're doing with the others. And the libfsverity.h include should be moved outside the group of rpm's own includes to have it stand out a bit.

Reply:
Fixed, thanks!